pytensor/link/c/type.py

```diff
@@ -505,9 +505,12 @@ def __delitem__(self, key):
     def __hash__(self):
         # All values are Python basic types, then easy to hash.
         return hash(
-            (type(self), self.ctype)
-            + tuple((k, self[k]) for k in sorted(self.keys()))
-            + tuple((a, self.aliases[a]) for a in sorted(self.aliases.keys()))
+            (
+                type(self),
+                self.ctype,
+                *((k, self[k]) for k in sorted(self.keys())),
+                *((a, self.aliases[a]) for a in sorted(self.aliases.keys())),
```

Comment:
This should also further simplify:

```suggestion
                *sorted(self.items()),
                *sorted(self.aliases.items()),
```

But I trust automated changes more than by-hand changes, so feel free to ignore this suggestion.